### ISSUE TITLE
Running jsonlint against '**/*.json' returns an error if no files found

### ISSUE THREAD

**Issue by isuftin** [prantlf/jsonlint#20]
```sh
❯ jsonlint --quiet --compact --continue --no-duplicate-keys '**/*.json'
no files found
❯ echo $?
1
```

It would be amazing if there were an option to not have this return an error and instead to just continue on its merry way. We are using this utility as part of a general linting workflow in GitLab and sometimes we may run the utility without any json files to lint. We'd like to have it return the "no files found" message but instead return a 0 exit code.

**Comment by prantlf** [comment]
Thank you for the idea! I'm going to add an argument `--succeed-with-no-files` for this in the upcoming release.

**Comment by prantlf** [comment]
Released with [14.1.0](https://github.com/prantlf/jsonlint/releases/tag/v14.1.0), committed in 923f37fd4361d80886f9bcd4787630c9b331574d.
